src/components/input/styles.tsx

```diff
@@ -76,6 +76,11 @@ export const StyledInput = styled.input<StyledInputProps>`
     color: ${theme.color.text.text03};
   }
 
+  &:-webkit-autofill {
+    box-shadow: 0 0 0 1rem ${theme.color.background.ui03} inset;
+    -webkit-text-fill-color: ${theme.color.text.text01};
+  }
```

Comment:
Does Firefox work?

Reply:
I couldn't force Firefox to do autofill on this field, but I added additional selector targeting Firefox. As per documentation - it should work